AndroidStudio/assignments/advanced/3rd/DialogAssignment/app/src/main/AndroidManifest.xml

```diff
@@ -2,20 +2,16 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="jp.mixi.assignment.dialog"
     android:versionCode="1"
-    android:versionName="1.0" >
-
-    <uses-sdk
-        android:minSdkVersion="8"
-        android:targetSdkVersion="17" />
+    android:versionName="1.0">
 
     <application
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/hidey/AndroidTraining/blob/2ec9edbd5ba91b173ceff2de43f76baf77b099b0/AndroidStudio/assignments/advanced/3rd/DialogAssignment/app/src/main/AndroidManifest.xml#L7"></span>App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent filter. See issue explanation for more details.</td>
    </tr>
  </tbody>
</table>